src/components/input/styles.module.css

```diff
@@ -0,0 +1,26 @@
+.inputContainer {
+    width: 70vw;
+    height: 20vh;
+    margin:1.5rem auto;
+    background-color: var(--card-bg);
+    box-shadow: var(--card-shadow-hover);
+    transition: box-shadow 0.2s ease-in-out, tranform 0.2s ease-in-out, background-color 0.5s ease-in-out;
+    border-radius: 0.5rem;
+}
+
+textarea {
```

Comment:
avoid styling the element as it is, use unique classes, there could be multiple textAreas from different components, and we don't want the same styles for all those textarea elements, similarly for button and other components too

Reply:
Done